control/statesp.py

```diff
@@ -91,38 +91,21 @@ class StateSpace(LTI):
 
     """
 
-    def __init__(self, *args):
+    def __init__(self,A,B,C,D,dt=None):
```

Comment:
This change will break previous code where users made use of the key word arg, e.g. `StateSpace(1, 2, 3, 4, dt=0.1)` will fail with a TypeError.

Reply:
I don't really know what you mean, you can still pass arguments as *args after the change, like this:

`args=(A,B,C,D)`
`StateSpace(*args)`
`args=(A,B,C,D,dt)`
`StateSpace(*args)`

The only thing the change will break is that you can no longer pass a StateSpace instance to `StateSpace.__init__` to create a new one, you'll have to be explicit. I think the work required to unpack the values is a reasonable tradeoff to the simplicity gained.

Reply:
```
In [6]: def f(*args):
   ...:     return args
   ...: 

In [7]: f(1, 2, 3, 4, dt=0.5)
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-7-ced6aff2bf09> in <module>()
----> 1 f(1, 2, 3, 4, dt=0.5)

TypeError: f() got an unexpected keyword argument 'dt'
```

Reply:
Your code is an example of the old behavior, that I removed.

This is the new behavior:

```
>>> def f(a,b,c,d,dt=None):
...     return a,b,c,d,dt
...     
>>> f(1,2,3,4,dt=None)
(1, 2, 3, 4, None)
```

Reply:
You are right. Dyslexia I suppose :)